### ISSUE TITLE
Blobs not found

### ISSUE THREAD

**Issue by vbar** [Blobscan/blobscan#453]
This isn't exactly the same as https://github.com/Blobscan/blobscan/issues/373 (the commitments as well as the error message are different), but from a user's perspective it's pretty much the same problem:

Going over blobs used by [zkSync Era](https://docs.zksync.io/build/) using [Blobscan API](https://api.blobscan.com/#/blobs/blob-getByBlobId), most of them can be retrieved w/o problems, but blobs of block 20083513 are missing:

```
$ curl https://api.blobscan.com/blobs/0x8225f8892f9609eca200d5f05116aaa076e6ce4822139a09709aade7190ee12336a12c1cbd4638127e0cb9a37b83c152
{"message":"No blob with versioned hash or kzg commitment '0x8225f8892f9609eca200d5f05116aaa076e6ce4822139a09709aade7190ee12336a12c1cbd4638127e0cb9a37b83c152'.","code":"NOT_FOUND"}
```

Same for 0x8c5cbc55845bb91e9a67eba27e4c80e69406f5794fee1f512f53847fecee6059cc6daad04424881307fbdd474dbb3399 and 0xb69196e0be224335a18e420e8be33693a3b1d674dac0245c081f4368fb235e1ac03b5d042aa292b2ab41dcd4dca61640 . The blobs are available on etherscan.io .

**Comment by PJColombo** [comment]
These blobs should be indexed now @vbar.

I'll let you know when we finish checking for any other missing block, tx or blob

**Comment by vbar** [comment]
The reported problem is fixed, a new one is in #604 :-)
